dist/notice/ds4/notice.css

```diff
@@ -319,19 +319,16 @@ button.page-notice__close span {
   background-image: url('data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIHZpZXdCb3g9IjAgMCAyNCAyNCI+PHBhdGggZmlsbD0iI2RkMWUzMSIgZD0iTTEyLjAwNyAxNC4yNjFhLjg0OS44NDkgMCAwIDEtLjg0OS0uODQ5VjYuODU5YS44NDkuODQ5IDAgMSAxIDEuNjg0LjAwNXY2LjU0OWEuODQ5Ljg0OSAwIDAgMS0uODM0Ljg0OWwtLjAwMS0uMDAxem0xLjA1NyAyLjg5NGExLjAxNiAxLjAxNiAwIDEgMS0yLjAzMiAwIDEuMDE2IDEuMDE2IDAgMCAxIDIuMDMyIDB6bS0xLjA1NyA2Ljg1OUM1LjM3NiAyNC4wMTQgMCAxOC42MzggMCAxMi4wMDcgMCA1LjM3NiA1LjM3NiAwIDEyLjAwNyAwYzYuNjMxIDAgMTIuMDA3IDUuMzc2IDEyLjAwNyAxMi4wMDd2LjAxNWMtLjAwOCA2LjYyNS01LjM4MSAxMS45OTItMTIuMDA3IDExLjk5MnptMC0yMi4yNjFjLTUuNjcxIDAtMTAuMjY4IDQuNTk3LTEwLjI2OCAxMC4yNjhzNC41OTcgMTAuMjY4IDEwLjI2OCAxMC4yNjggMTAuMjY4LTQuNTk3IDEwLjI2OC0xMC4yNjhTMTcuNjc4IDEuNzUzIDEyLjAwNyAxLjc1M3oiLz48L3N2Zz4=');
   height: 24px;
   width: 24px;
-  background-size: contain;
```

Comment:
Should this be missing in notice?

Reply:
I believe that change to notice is correct, since the height and width are over 24px.